Changes

```diff
@@ -1,5 +1,15 @@
 Unreleased
 
+Release 2.46.0
+
+  - Mark tests on iRODS 4.3.1 as optional
+  - Add iRODS 4.3.1 Ubuntu 22.04 as a required test target
```

Comment:
"a required" -> "an optional"

Reply:
Should be done now. Adding one more commit but will squash and merge if you think it looks ok now.